casanovo/denovo/model_runner.py

```
@@ -289,6 +289,15 @@ def predict(
         self.initialize_model(train=False)
         self.model.out_writer = self.writer
 
+        if "max_peaks" in self.model.hparams:
+            ckpt_max_peaks = self.model.hparams["max_peaks"]
+            if self.config.max_peaks != ckpt_max_peaks:
+                logger.warning(
+                    f"Overriding config max_peaks ({self.config.max_peaks}) "
+                    f"with checkpoint max_peaks ({ckpt_max_peaks})"
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)